README.md

```diff
@@ -341,20 +341,20 @@ $ CONFIGMAP_GENERATOR_POD=`oc get pods | grep "httpd-configmap-generator" | cut
 ### Generating a configmap for external authentication against IPA
 
 ```
-$ oc rsh $CONFIGMAP_GENERATOR_POD httpd_configmap_generator ipa ...
+$ oc exec $CONFIGMAP_GENERATOR_POD  -- bash -c 'httpd_configmap_generator ipa ...
 ```
 
 Example configuration:
 
 ```
-$ oc rsh $CONFIGMAP_GENERATOR_POD httpd_configmap_generator ipa \
+$ oc rsh $CONFIGMAP_GENERATOR_POD -- bash -c 'httpd_configmap_generator ipa \
```

Comment:
oc exec ... ?

Reply:
@abellotti, do you want me to replace rsh cmd with exec one ?
exec command  doesn't read bash config/profile by default and requires bash calls

Reply:
👍 LGTM @abellotti I believe this can and should be merged.

Reply:
@jvlcek, done, thanks.